taf/api/targets.py

```diff
@@ -662,3 +662,83 @@ def _update_target_repos(
         path = targets_dir / target_repo_name
         path.write_text(json.dumps(data, indent=4))
         taf_logger.info(f"Updated {path}")
+
+
+@check_if_clean
+def taf_status(path: str, library_dir: Optional[str] = None) -> None:
```

Comment:
In our system, we have an authentication and its target repositories, but also the concept of dependencies. Dependencies are other authentication repositories which are linked with the current authentication repository. This is specified in a file called `dependencies.json`. Just like we have `load_repositories` that is used here, we also have `load_depndencies` and you can then interate over them `child_auth_repos= repositoriesdb.get_deduplicated_auth_repositories( auth_repo, commits).values()`

Then, you'd need to call this recursively. But we want to show clearly which repository is the parent, so we want to indent properly. Since list targets already outputs a json, then we can add information about the current repository and if it has dependencies, add their information under the `dependencies` key. It's very unlikely that we'll have deep nestings.